alignak_module_nrpe_booster/nrpe_booster.py

```diff
@@ -223,7 +217,7 @@ def __init__(self, host, port, use_ssl, timeout, unknown_on_timeout, msg):
 
         self.create_socket(addrinfo[0], socket.SOCK_STREAM)
 
-        if use_ssl:
+        if self.use_ssl:
             # The admin want a ssl connection,
             # but there is not openssl lib installed :(
```

Comment:
I think we can remove this code because openssl installed with alignak by default